### PR TITLE
Renamed the existing Supported Platforms section

### PR DESCRIPTION
**Pull request by nidhibhammar**
## What Changed?

Renamed the existing Supported Platforms section and added the Hardware Requirements and Software prerequisites to the same section.

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [X] This PR adds new content
- [X] This PR changes existing content
- [X] This PR removes existing content
- [ ] This PR is for a release, please add this tag:

### CODE REVIEW

File: product_docs/docs/epas/14/epas_platform_support/index.mdx [hunk 1]
```diff
@@ -18,6 +20,28 @@ EDB Postgres Advanced Server v14 Beta 1 supports installations on the following
 
 See [Product Compatibility](https://www.enterprisedb.com/platform-compatibility) for additional information.
 
+## Hardware Requirements
+
+The following installation requirements assume you have selected the default options during the installation process. The minimum hardware required to install and run EDB Postgres Advanced Server are:
```
Comment (drothery-edb) [inline product_docs/docs/epas/14/epas_platform_support/index.mdx:25]:
```suggestion
The following installation requirements assume you select the default options during the installation process. The minimum hardware requirements to install and run EDB Postgres Advanced Server are:
```

File: product_docs/docs/epas/14/epas_platform_support/index.mdx [hunk 2]
```diff
@@ -18,6 +20,28 @@ EDB Postgres Advanced Server v14 Beta 1 supports installations on the following
...
+
+Additional disk space is required for data or supporting components.
+
+## Software Prerequisites
+
+**User Privileges**
+
+To perform an Advanced Server installation on a Windows system you must have administrator privilege. If you are installing Advanced Server on a Windows system that is configured with `User Account Control` enabled, you can assume sufficient privileges to invoke the graphical installer by right clicking on the name of the installer and selecting `Run as administrator` from the context menu.
+
+To perform an Advanced Server installation on a Linux system you must have superuser or administrator or sudo privileges.
```
Comment (drothery-edb) [inline product_docs/docs/epas/14/epas_platform_support/index.mdx:39]: Since Linux is our primary platform, move the Linux paragraph above the Windows one.

Comment (drothery-edb) [reply product_docs/docs/epas/14/epas_platform_support/index.mdx:39]: Also, be consistent with the EPAS product name, we are switching between the full name and the short name in this topic.

**Review by drothery-edb — APPROVED** [review]
I left a few comments

File: product_docs/docs/epas/14/epas_platform_support/index.mdx [hunk 3]
```diff
@@ -34,9 +34,9 @@ Additional disk space is required for data or supporting components.
 
 **User Privileges**
 
-To perform an Advanced Server installation on a Windows system you must have administrator privilege. If you are installing Advanced Server on a Windows system that is configured with `User Account Control` enabled, you can assume sufficient privileges to invoke the graphical installer by right clicking on the name of the installer and selecting `Run as administrator` from the context menu.
+To perform an EDB Advanced Server installation on a Linux system you must have superuser or administrator or sudo privileges.
```
Comment (drothery-edb) [inline product_docs/docs/epas/14/epas_platform_support/index.mdx:37]: use either the full name (EDB Postgres Advanced Server) or short name (Advanced Server). Best practice, if using the short name) is to introduce it in the first instance. For example, EDB Postgres Advanced Server (Advanced Server)

Comment (nidhibhammar) [reply product_docs/docs/epas/14/epas_platform_support/index.mdx]: Done the changes

**Review by drothery-edb — DISMISSED** [review]

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**